.gitignore

```diff
@@ -5,3 +5,8 @@ bin/terraform*
 platform-tests/pkg
 platform-tests/src/results
 terraform/.terraform/
+node_modules/
+package-lock.json
```

Comment:
IMO lockfiles should be checked in to git

Reply:
But there shouldn't be a `package-lock.json` or `package.json` at all in the project. Should it?

Reply:
it does seem weird this is at the root of the repository. Perhaps a sub folder for org management or something?

Reply:
As it is only there for running the contact_orgs.js script, I decided to add it as an exemption. @paroxp has stated that this should be a feature in Pazmin, and as a result it is only short lived.

Reply:
I'd remove the `package-lock.json` from `.gitignore` to avoid potential problems in the future.

Hector's right. We're not using `package.json` file - which may be a problem as I imagine people don't just have `notifications-node-client` npm package installed globally, nor is it documented that it's required (I think) - therefore it won't work out of the box sadly.

As Lee mentioned, this should some day become a feature of paas-admin - but we should make sure it works for people who might be using it.